src/pages/Customer/Auth/Verification/Verification.jsx

```diff
@@ -1,29 +1,113 @@
-import '../../../../components/Auth/Forms/global.module.scss'
+import React, { useState } from 'react'
+import { useLocation, useNavigate } from 'react-router-dom'
 
-import { Row } from 'antd'
-import React from 'react'
+import '../../../../components/Auth/Forms/global.module.scss'
+import { Row, Form, Input } from 'antd'
 
-import VerificationForm from '../../../../components/Auth/Forms/Verification/Verification'
+import { useCustomerVerificationCodeMutation } from '../../../../features/slices/auth/customer/customerAuthApiSlice'
 import FormDescription from '../../../../components/Auth/Forms/Widgets/FormDescription'
 import PageIndicator from '../../../../components/Auth/Forms/Widgets/FormPageIndicator'
 import imageDesc from '../../../../../src/assets/Auth/Analyze-amico 1.svg'
 import LeftLayout from '../../../../components/Auth/Layout/LeftLayout/LeftLayout'
 import RightLayout from '../../../../components/Auth/Layout/RightLayout/RightLayout'
 import classes from './Verification.module.scss'
-
+import FormButton from '../../../../components/Auth/Forms/Widgets/FormButton'
+import FormHeader from '../../../../components/Auth/Forms/Widgets/FormHeader'
 const Verification = () => {
+  const email = useLocation()
   const formDescription = {
     image: imageDesc,
     header: 'Graphs and charts',
     tagline:
       'Have a better experience irrespective of the device, OS, screen size, orientation, and browser platform.',
     ImgHeight: '2px',
   }
+  const [errMsg, setErrMsg] = useState('')
+  const [customerVerificationCode, { isLoading }] =
+    useCustomerVerificationCodeMutation()
+  const navigate = useNavigate()
+  const onFinish = async (values) => {
+    try {
+      await customerVerificationCode({
+        email: email.state.email,
+        otp: values.otp,
+      }).unwrap()
+      navigate('/business')
+    } catch (err) {
+      console.log(err)
+      if (err.status === 401) {
```

Comment:
I see this block of code a lot, don't you think extracting it to a function and reusing it would be easier
```
if (err.status === 401) {
        setErrMsg(err?.data?.detail)
      } else if (err.status === 400) {
        setErrMsg(err?.data?.email)
      } else if (err.status === 500) {
        setErrMsg('Cannot connect to server.')
      } else {
        setErrMsg('Check your internet connection')
      }
```

Reply:
I would, i'd also span across my scss files, thanks.